### PR TITLE
feat: support for `screen_hint` on authorize

### PR DESCRIPTION
**Pull request by danestves**
## Description

Add support for `screen_hint` and also document every option for the developer to know what is happening with that function

## Motivation and Context

Provide a user a way to show directly the signup page instead of the login

### CODE REVIEW

**Review by meza — CHANGES_REQUESTED** [review]

**Comment by meza** [comment]
I fixed the actions issue on the `next` branch and changed the PR target to be the next too. If you bring that change in, it should pass too. Seems like one of the actions needed a token.

File: src/index.ts
[the remark this answers is not in the document]
Comment (danestves) [reply src/index.ts:1]: Done!

**Comment by danestves** [comment]
Changes made and made some changes on documentation too

**Pull request merged**